### PR TITLE
[env] Introduce JvmUtils to support global JNIEnv

### PR DESCRIPTION
**Pull request by ljz2051**
## What is the purpose of the change

This pull request introduce a JvmUtil to support global JNIEnv.

## Brief change log

- Add JvmUtil class for flink env.

## Verifying this change

New tests will be introduced after FlinkEnv's JNI interfaces are supported

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1016,7 +1016,8 @@ else()
     port/port_posix.cc
     env/env_posix.cc
     env/fs_posix.cc
-    env/io_posix.cc)
+    env/io_posix.cc
+          env/flink/JavaUtil.cc)
```
Comment (masteryhx) [inline CMakeLists.txt]: How about naming java_util.cc which is consistent with other files ?

File: java/rocksjni/env_flink.cc [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```
Comment (masteryhx) [inline java/rocksjni/env_flink.cc]: Is this ready for this pr?

**Review by masteryhx — CHANGES_REQUESTED** [review]
Thanks for the pr.
The CI failed and you could try 'make format' firstly.

File: CMakeLists.txt (hunk 1 above)
Comment (ljz2051) [reply CMakeLists.txt]: This class has been renamed to jvm_util.cc

File: java/rocksjni/env_flink.cc (hunk 2 above)
Comment (ljz2051) [reply java/rocksjni/env_flink.cc]: I have removed this file in the latest commit. The JNI support of FlinkEnv will be introduced in another pr.

**Review by masteryhx — APPROVED** [review]
LGTM.

**Pull request merged**